reown-com/appkit
# add universal link flag

**TODO**
- [x] Filter wallets if they don't support Universal Link
- [ ] Fix iOS issue

## How to test the preview in Telegram:

1. Look for the bot called @appkit_labs_bot in Telegram
2. Send any message or press start.
3. Labs will be open, choose the "wagmi default" option
4. Press the connect button, it should redirect to the wallet (any wallet) and be able to connect


# Changes

- feat: Add `enableUniversalLinks` flag to use Universal Linking on mobile, this will improve [Telegram Mini Apps](https://docs.ton.org/develop/dapps/telegram-apps/) support

Blocked by https://github.com/WalletConnect/cloud-app/pull/1195

# Associated Issues

closes #...
